js/templates/search/AddProvider.html

```diff
@@ -0,0 +1,12 @@
+<div class="tx5 confirmBox arrowBoxCenteredTop padMd clrBr clrP clrSh1 clrT addBox js-addProvider <% if (ob.hideBox) print('hide') %>">
+  <div class="flexCol gutterV txLft">
+    <h3><%= ob.polyT('search.addTitle') %></h3>
+    <% const plhdr = ob.polyT(`${ob.usingTor ? 'search.addTorPlaceholder' : 'search.addPlaceholder'}`) %>
+    <% if (ob.errors['listings']) print(ob.formErrorTmpl({ errors: ob.errors['listings'] })) %>
```

Comment:
I think you would want to pass in `usingTor` into here and if it's true show the `torListings` error if it exists instead of the `listings` error.

Reply:
Old copy and past again.

Reply:
Not actually copy and paste, see earlier comments. This has been refactored to use separate url errors based on a passed in url type (since there are 4 possible urls currently).